test/framework/resources/aws/services/iam.go

```diff
@@ -84,6 +85,13 @@ func (d *defaultIAM) GetInstanceProfile(instanceProfileName string) (*iam.GetIns
 	return d.IAMAPI.GetInstanceProfile(getInstanceProfileInput)
 }
 
+func (d *defaultIAM) ListPolicies(scope string) (*iam.ListPoliciesOutput, error) {
+	listPolicyInput := &iam.ListPoliciesInput{
+		Scope: aws.String(scope),
+	}
+	return d.IAMAPI.ListPolicies(listPolicyInput)
```

Comment:
should we return errors here as well ? https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListPolicies.html#API_ListPolicies_ResponseElements

Reply:
iam interface ListPolicies has two return values so this should work as is: 
https://pkg.go.dev/github.com/aws/aws-sdk-go@v1.40.6/service/iam/iamiface#IAMAPI.ListPolicies